app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/GetServiceRequestsController.php

```diff
@@ -86,4 +87,45 @@ public function __invoke(): JsonResponse
             )
         );
     }
+
+    public function show(ServiceRequest $serviceRequest): JsonResponse
+    {
+        $license = resolve(LicenseSettings::class);
+        $settings = resolve(PortalSettings::class);
+        $contact = auth('contact')->user();
+
+        $enabled = $license->data->addons->serviceManagement
+            && $settings->knowledge_management_portal_service_management
+            && $contact;
+
+        if (! $enabled) {
+            return response()->json();
+        }
+
+        return response()->json([
+            'serviceRequestDetails' => [
+                'id' => $serviceRequest->getKey(),
+                'title' => $serviceRequest->title,
+                'description' => $serviceRequest->close_details,
+                'serviceRequestNumber' => $serviceRequest->service_request_number,
+                'statusName' => $serviceRequest->status?->name,
+                'statusColor' => $serviceRequest->status?->color->value,
+                'typeName' => $serviceRequest->priority?->type?->name,
+            ],
+            'serviceRequestUpdates' => $serviceRequest
+                ->serviceRequestUpdates()
+                ->latest('created_at')
+                ->get()
+                ->map(function (ServiceRequestUpdate $serviceRequestUpdate) {
+                    return [
+                        'id' => $serviceRequestUpdate->getKey(),
+                        'update' => $serviceRequestUpdate->update,
+                        'internal' => $serviceRequestUpdate->internal,
+                        'direction' => $serviceRequestUpdate->direction,
+                        'created_at' => $serviceRequestUpdate->created_at->format('m-d-Y g:i A'),
+                    ];
+                })
+                ->toArray(),
```

Comment:
We also want to make sure that on the Front End, the color difference between updates is based on whether it is inbound or outbound instead of internal/external since all will be external now.

Reply:
Done